components/leds.py

```diff
@@ -125,40 +136,87 @@ def morse(self) -> Tuple[int, int, int]:
                 if token == " ":
                     return LedColours.OFF.value
                 else:
-                    return self.colour
+                    return LedColours.BLUE.value
         # Default - should never be hit
         return LedColours.OFF.value
 
-    def choose_morse_message(self, message: str = None) -> None:
+    def _pacman_calculation(self) -> None:
+        elapsed_time = time.monotonic() - self.pattern_start_time
+        # find the pattern of leds and its position
+        if elapsed_time % self.PACMAN_PERIOD < self.PACMAN_PERIOD / 2:
```

Comment:
Isn't `elapsed_time % self.PACMAN_PERIOD == elapsed_time` always due to the if statement on line 203?